odbc/src/api/functions.rs

```diff
@@ -1785,11 +1785,441 @@ pub unsafe extern "C" fn SQLGetInfo(
+            //  - MongoSQL's grammar defines an identifier character as [^\x00], as in, literally any non-null character.
+            //    Surely we aren't meant to return a "special characters" string that contains every character, are we?
+        }
+        // SQL_MAX_COLUMNS_IN_GROUP_BY
+        InfoType::MaxColumnsInGroupBy => {
+            // MongoSQL does not have an explicit maximum number of
+            // columns allowed in a GROUP BY clause.
+            return i16_len::set_output_fixed_data(&SQL_U16_ZERO, info_value_ptr, string_length_ptr);
+        }
+        // SQL_MAX_COLUMNS_IN_ORDER_BY
+        InfoType::MaxColumnsInOrderBy => {
+            // MongoSQL does not have an explicit maximum number of
+            // columns allowed in a ORDER BY clause.
+            return i16_len::set_output_fixed_data(&SQL_U16_ZERO, info_value_ptr, string_length_ptr);
+        }
+        // SQL_MAX_COLUMNS_IN_SELECT
+        InfoType::MaxColumnsInSelect => {
+            // MongoSQL does not have an explicit maximum number of
+            // columns allowed in a SELECT clause.
+            return i16_len::set_output_fixed_data(&SQL_U16_ZERO, info_value_ptr, string_length_ptr);
+        }
+        // SQL_TIMEDATE_ADD_INTERVALS
+        InfoType::TimedateAddIntervals
+        // SQL_TIMEDATE_DIFF_INTERVALS
+        | InfoType::TimedateDiffIntervals => {
+            // MongoSQL DATEADD and DATEDIFF support the following intervals.
+            const TIMEDATE_INTERVALS: u32 = SQL_FN_TSI_SECOND
+                | SQL_FN_TSI_MINUTE
+                | SQL_FN_TSI_HOUR
+                | SQL_FN_TSI_DAY
+                | SQL_FN_TSI_WEEK
+                | SQL_FN_TSI_MONTH
+                | SQL_FN_TSI_QUARTER
+                | SQL_FN_TSI_YEAR;
+            return i16_len::set_output_fixed_data(&TIMEDATE_INTERVALS, info_value_ptr, string_length_ptr);
+        }
+        // SQL_CATALOG_LOCATION
+        InfoType::CatalogLocation => {
+            // MongoSQL puts the catalog (database) at the start of a qualified
+            // table name. As in, db.table.
+            return i16_len::set_output_fixed_data(&SQL_CL_START, info_value_ptr, string_length_ptr);
+        }
+        // SQL_SQL_CONFORMANCE
+        InfoType::SqlConformance => {
+            // MongoSQL is SQL-92 Entry level compliant.
+            return i16_len::set_output_fixed_data(
+                &SQL_SC_SQL92_ENTRY,
+                info_value_ptr,
+                string_length_ptr,
+            );
+        }
+        // SQL_ODBC_INTERFACE_CONFORMANCE
+        InfoType::OdbcInterfaceConformance => {
+            // The MongoSQL ODBC Driver is currently meets the minimum compliance level.
+            return i16_len::set_output_fixed_data(&SQL_OIC_CORE, info_value_ptr, string_length_ptr);
+        }
+        // SQL_SQL92_PREDICATES
+        InfoType::Sql92Predicates => {
+            // MongoSQL supports the following SQL-92 predicate operators.
```

Comment:
We also support DISTINCT and EXISTS, so I would add SQL_SP_UNIQUE and SQL_SP_EXISTS, no?

Reply:
What is the specific SQL-92 predicate expression that `SQL_SP_UNIQUE` corresponds to? I cannot think of any predicates we have in MongoSQL that would match that.

Also, I have `SQL_SP_EXISTS` first in the list!

Reply:
ah, I think you are right here :) All good to go 👍